### PR TITLE
Implement Respond Reminders

### PR DESCRIPTION
**Pull request by Teddarific**

### CODE REVIEW

File: app/package.json [hunk 1]
```diff
@@ -138,6 +138,7 @@
     "framer-motion": "^6.5.1",
     "fs": "0.0.1-security",
     "node-machine-id": "^1.1.12",
+    "node-notifier": "^10.0.1",
```
Comment (Teddarific) [inline app/package.json:141]: Remove this

File: app/src/main/LiveDb/getRespondReminders.ts [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+function scoreReminder(reminder: RespondRemindersResult) {
+  // Longer is good
+  // Includes ? is good
+  // Contact is legit is good
+  return (
+    reminder.message.length +
+    300 *
+      (reminder.message.includes('?') ? 2 : 1) *
+      (hasNumber(reminder.friend) ? 1 : 10)
```
Comment (Teddarific) [inline app/src/main/LiveDb/getRespondReminders.ts:19]: Let's maybe share this logic with the other one too

File: app/src/analysis/directories.ts [hunk 3]
```diff
@@ -18,3 +18,21 @@ export const chatPaths = {
...
+export const appDirectoryLivePath = `${process.env.HOME}/.leftonread/live`;
+
+export const liveAddressBookPaths = {
+  original: `${process.env.HOME}/Library/Application Support/AddressBook`,
+  app: `${appDirectoryLivePath}/AddressBookFolder`,
+};
+
+export const liveChatPaths = {
+  original: process.env.DEBUG
+    ? `./src/__tests__/chat.db`
```
Comment (alexdanilowicz) [inline app/src/analysis/directories.ts:31]: The `./src/__tests__/chat.db` is busted right now btw. Whenever it goes to git, it gets corrupted.

File: app/src/utils/amplitudeClient.ts [hunk 4]
```diff
@@ -1,8 +1,33 @@
...
       };
+
+export function logEventMain({
+  eventName,
+  properties,
+}: {
+  eventName: string;
+  properties?: Record<string, string | number>;
+}) {
+  const uuid = getUuid();
```
Comment (alexdanilowicz) [inline app/src/utils/amplitudeClient.ts:20]: ~should we store this in the electron store?~

EDIT: you already do. nice!

**Review by alexdanilowicz — COMMENTED** [review]
bIg BrAin 🧠 🚀

**Pull request merged**